snap/squashfs/squashfs.go

```diff
@@ -90,6 +90,14 @@ func (s *Snap) Install(targetPath, mountDir string) error {
 
 var runCommandWithOutput = func(args ...string) ([]byte, error) {
```

Comment:
I think this is the wrong place for switching LANG, this is a generic "runCommandWithOutput" helper. This side-effect is surprising for most people. I think instead we should set/change the environment inside the tests only